README.md

```diff
@@ -40,49 +46,145 @@ SpaceDocks build process follows the golang specifications: You need to clone th
 ---- sdb.go
 ```
 
-#### Glide
-We use the [Glide package manager](https://glide.sh) to manage the sdb dependencies. For installing glide, please execute the appropreate buildscript for your plattform (Linux/MacOS Users: build/install_glide.sh, Windows Users: build/install_glide.ps1). This will install glide into $GOPATH/bin, which requires a valid $GOPATH! It is also **required** to add $GOPATH/bin to your $PATH!
+### Get the dependencies
+
+#### Install Go
+
+Install Go 1.8. You can get it from https://golang.org/dl/ or from apt-get:
+
+```sh
+apt-get install golang
```

Comment:
I am not sure if the `golang` package is what we want. It installes an up-to-date version for newer ubuntu releases, but for example in 16.04 LTS it installs Go 1.6, which is definitly too old to run SDB IIRC.

`apt install golang-1.9` would install the same version on all releases of Ubuntu (and possibly Debian too)